.ci/test_r_package_windows.ps1

```diff
@@ -0,0 +1,110 @@
+# Download a file and retry upon failure. This looks like
+# an infinite loop but CI-level timeouts will kill it
+function Download-File-With-Retries {
+  param(
+    [string]$url,
+    [string]$destfile
+  )
+  do {
+    Write-Output "Downloading ${url}"
+    sleep 5;
+    (New-Object System.Net.WebClient).DownloadFile($url, $destfile)
+  } while(!$?);
+}
+
+$env:R_WINDOWS_VERSION = "3.6.3"
+$env:R_LIB_PATH = "$env:BUILD_SOURCESDIRECTORY/RLibrary" -replace '[\\]', '/'
+Write-Output "R_LIB_PATH: $env:R_LIB_PATH"
```

Comment:
I guess this line was accidentally left after debugging, wasn't it?

Reply:
oh sure, can be removed

Reply:
removed in https://github.com/microsoft/LightGBM/pull/2936/commits/747378eb41b74d5124b905f9cca5f76727d3a5ae